csharp/src/ProtocolBuffers.Test/CodedInputStreamTest.cs

```diff
@@ -232,66 +230,26 @@ public void DecodeZigZag64()
             Assert.AreEqual(0x7FFFFFFFFFFFFFFFL, CodedInputStream.DecodeZigZag64(0xFFFFFFFFFFFFFFFEL));
             Assert.AreEqual(unchecked((long) 0x8000000000000000L), CodedInputStream.DecodeZigZag64(0xFFFFFFFFFFFFFFFFL));
         }
-        /*
+        
         [Test]
-        public void ReadWholeMessage()
+        public void ReadWholeMessage_VaryingBlockSizes()
         {
-            TestAllTypes message = TestUtil.GetAllSet();
+            TestAllTypes message = GeneratedMessageTest.GetSampleMessage();

```

Comment:
Okay, that's fine by me. Given the number of PRs in the "train" now, do you mind if I do that after we've merged the PRs? Doing a lot of rebasing now for something that can easily be done later (probably in a complete separate PR) feels error-prone, at least with my level of git competence.